src/list.txt

```diff
@@ -0,0 +1,58 @@
+fonduer.candidates.candidates
```

Comment:
What's the purpose of this file?

Reply:
Oh, that was sort of TODO list. My bad.
Also I'll resolve the conflict.